content/Arduino Cloud/Arduino IoT Cloud/If-your-device-cant-be-added-or-wont-connect-to-IoT-Cloud.md

```diff
@@ -121,7 +109,7 @@ This is a generic error message indicating that a process has failed.
 
```

Comment:
The step 2 in "If the sketch uploads but the device isn't online" has some typos. Suggestion: "Go to the [Things tab](https://cloud.arduino.cc/iot/things) and open the Thing that's associated to your device. Then go to the [Devices tab](https://cloud.arduino.cc/iot/devices), and find your device in the list. Under Network, click on Change and verify that your credentials are correct."

Reply:
891884730fbc3c640f1cfd295da2ae673d3c68bc